dao/src/main/resources/io/syndesis/dao/deployment.json

```diff
@@ -1,1897 +1,2625 @@
 [
-  {
```

Comment:
Good catch, my editor sucks :( and reformatted the entire file even though I only highlighted the added section. Should be fixed now.